framework/helpers/BaseHtml.php

```diff
@@ -1157,6 +1157,18 @@ public static function activeInput($type, $model, $attribute, $options = [])
      */
     public static function activeTextInput($model, $attribute, $options = [])
     {
+        if (!isset($options['maxlength'])) {
```

Comment:
@samdark I assumed that the user would never manual type any value different than the string max value.
@cebe If the option isn't created, the input is normally rendered. Any value with the key `maxlength` will try to the find the `max` (perhaps make here a explicity value (e.g. `true`))